packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/add/add.controller.js

```diff
@@ -41,33 +41,9 @@ export default class {
   }
 
   create() {
-    this.isAdding = true;
-    return this.Kubernetes.createNodePool(this.projectId, this.kubeId, {
-      desiredNodes: this.nodePool.nodeCount,
-      flavorName: this.nodePool.flavor.name,
-      name: this.nodePool.name,
-      antiAffinity: this.nodePool.antiAffinity,
-      monthlyBilled: this.nodePool.monthlyBilling,
-    })
-      .then(() =>
-        this.goBack(
-          this.$translate.instant('kube_add_node_pool_success', {
-            nodePoolName: this.nodePool.name,
-          }),
-          'success',
-        ),
-      )
-      .catch((error) => {
-        this.CucCloudMessage.error(
-          this.$translate.instant('kube_add_node_pool_error', {
-            nodePoolName: this.nodePool.name,
-            message: get(error, 'data.message', error.message),
-          }),
-        );
-      })
-      .finally(() => {
-        this.isAdding = false;
-      });
+    this.trackClick(
+      'PublicCloud::pci::projects::project::kubernetes::details::nodepools::add::cancel',
```

Comment:
Is this correct? shoul'nt it be create/confirm instead of cancel?

Reply:
True @radireddy.
Thanks.

Reply:
Fixed.